lib/transport_web/templates/dataset/details.html.eex

```diff
@@ -44,17 +44,26 @@
 
       <div class="dataset-details__discussions">
         <h2>Discussions</h2>
-        <discussions
-          datasetid="<%= @dataset.id %>"
-          site="<%= @site %>"
-          connected="<%= @current_user != nil %>"
-          posted_on="<%= dgettext("page-dataset-details", "Posted on") %>"
-          respond_comment="<%= dgettext("page-dataset-details", "Respond") %>"
-          post_discussion="<%= dgettext("page-dataset-details", "Begin a new conversation") %>"
-          title="<%= dgettext("page-dataset-details", "Title") %>"
-          connection_needed="<%= dgettext("page-dataset-details", "You need to be connected to post comments")%>"
-          type="null">
-        </discussions>
+        <%= for discussion <- @discussions do %>
```

Comment:
it can me rewritten as:
```elixir
<%= render_many @discussions, Transport.DatsetView, "_discussion.html", as: :discussion, current_user: @current_user, conn: @conn %>
```

Reply:
thanks! didn’t know about that one